apps/docs-website/eslint.config.ts

```diff
@@ -52,6 +52,7 @@ export default tseslint.config(
           },
         },
       ],
+      'import-x/no-unresolved': 0, // Doesn't know about Webpack aliases
```

Comment:
Webpack aliases have nothing to do with this (not sure where you got this from).

The reason why 'import-x/no-unresolved' is turned on, is because you turned it on in this PR, with these lines:

```
 extends: [
        pluginImportX.flatConfigs.recommended,
        pluginImportX.flatConfigs.typescript,
 ],
```

Just remove the presents and you can then remove this line.

Reply:
Docusurus uses webpack aliases, which `no-unresolved` can't handle.